src/Game.js

```diff
@@ -115,15 +104,11 @@ Game.prototype.runGame = function(currentPlayer) {
   }
 
   if (!this.gameIsWon && !this.gameIsDrawn) {
-    move = currentPlayer.getMove(this.gameBoard);
+    move = currentPlayer.getMove(currentPlayer.side);
```

Comment:
perhaps currentPlayer.getSide() instead of direct access to currentPlayer's property.